NEWS

```diff
@@ -11,6 +11,241 @@
+#### Core
+ * Fixed various typos
+#### CUDA
+ * Fixed async memory handling using cuda memory type on Grace
+ * Added rcache overhead in protocol estimation
+ * Fixed GDR copy performance regression by providing maximum estimation between get and put
+ * Fixed CUDA IPC reachability check
+ * Fixed crash in MPI_Finalize when CUDA context is destroyed
+ * Always require rcache by default for GDR
+ * Fixed md_close when no rcache
+ * Fixed CUDA copy memory domain allocations
+ * Fixed multiple tests for gdr_copy transport
+#### GO
+ * Fixed various issues
+ * Added serialized thread mode to avoid subtle races between threads
+ * Fixed make distcheck.
+#### RDMA CORE (IB, ROCE, etc.)
+ * Fixed FETCH_ADD remote access error for ODP/KSM case
+ * Fixed missing conditional compilation checks for DM
+ * Fixed IB MD allocation naming typo
+ * Fixed invalid GIDs filter in IB
+ * Fixed flags usage in MLX5 zcopy_post
+ * Do not limit ODP registration retries
+ * Fixed JUCX failures by considering the number of supported completion vectors
+#### ROCM
+ * Fixed ROCM interface reachability test
+ * Fixed memory domain fork test
+#### TCP
+ * Always bind endpoint to interface
+#### Tests
+ * Fixed core dump issue upon exception in gtest initialization
+ * Removed warning during configure file generation
+ * Fixed wire compatibility test failures
+ * Fixed test_callbackq.test_callbackq failures
+ * Fixed TCP device name detection in pci_bw test
+ * Fixed false positive coverity storage leak warning
+ * Fixed wrong library used by loader
+ * Fixed test by waiting Wait for flush to complete between puts
+ * Fixed transfer timeouts for BF
+ * Disabled RMA tests with protov1
+ * Skipped gdr_copy performance check in test_uct_loopback
+ * Fixed out-of-memory test failure by limiting MEMIC allocation
+ * Fixed heap-buffer overflow in AMO test
+ * Fixed tests by increasing UD EP timeout when running under valgrind
+#### Tools
+ * Fixed buffer size potential overflow in ucx_perftest
+ * Fixed missing address when packing memory keys
+ * Fixed memory leak for endpoint report in ucx_info
+ * Fixed build without openmp
+ * Fixed UCT device override on server side
+#### UCM
+ * Fixed race condition in parsing proc maps
+ * Fixed mremap failure while parsing /proc/self/maps
+#### UCP
+ * Fixed stack overflow in exported rkey unpack
+ * Removed extra remote-cpu overhead from protocol estimation for zcopy
+ * Fixed performance estimation for rndv pipeline protocols
+ * Fixed ATP sending by picking the correct lane
+ * Fixed missing reg_id on memh creation
+ * Fixed repeated invalidations by retaining existing access flags
+ * Fixed abort reason propagation for rendezvous RTR mtype
+ * Fixed transport availability for configuration list in ALLOW mode
+ * Fixed wrong flag being used for checking BCOPY capability
+ * Fixed sending too many ATPs for small messages
+ * Forced using active message identifiers within 16 bits
```

Comment:
pls rephrase

Reply:
wbf "Enforced 16 bits size for active messages identifiers"